.github/workflows/ci.yml

```
@@ -108,8 +108,8 @@ jobs:
           flags: unit-core
           name: codecov-unit-core
 
-  test-project-tools-scaffolding:
-    name: Project Tools Regression (Scaffolding)
+  test-project-tools-scaffold-core:
+    name: Project Tools: Scaffold Core
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)